Slam/Queue/QueueManagerProtocol.swift

```diff
@@ -0,0 +1,6 @@
+public protocol QueueManager {
+    var queueView: QueueView { get }
+
+    func loadQueue()
+    func reloadQueue(callback: () -> Void)
```

Comment:
This is probably fine for now, but this is an inconsistent pattern from what we are using in the "add match to queue" flow.

For now, I would rename `callback` to `onSuccess` or something to indicate that this is triggered was the queue was successfully loaded.